package.json

```diff
@@ -11,7 +11,34 @@
   "dependencies": {
     "axios": "^0.20.0",
     "discord.js": "^12.3.1",
+    "dotenv": "^8.2.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.22.1",
```

Comment:
can you move these lines to `devDependencies`? `devDependencies` are dependencies that you need as a developer, but that you don't need to run the code. By moving it to `devDependencies`, you avoid Render having to install these 2